binary_search/C/binary_search.c

```diff
@@ -0,0 +1,27 @@
+#include <stdio.h>
+
+int binarySearch(int arr[], int start, int end, int item)
+{
+	int mid = start + (end-start) / 2;
+	while (start <= mid) {
+		if (arr[mid] == item) {
+			return mid;
+		} else if (arr[mid] > item) {
+			end = mid-1;
+		} else {
+			start = mid+1;
+		}
+		mid = start + (end-start) / 2;
+	}
+	return -1;
+}
+
+int main()
+{
```

Comment:
{ should almost always be at the end of the previous line  [whitespace/braces] [4]

*Origin: CPPLintBear, Section: `all.cpplint`.*